src/Voron/Data/Tables/Table.cs

```diff
@@ -619,6 +619,32 @@ public IEnumerable<SeekResult> SeekForwardFrom(TableSchema.SchemaIndexDef index,
             }
         }
 
+        public long GetNumberEntriesFor(TableSchema.FixedSizeSchemaIndexDef index, long afterValue, out long totalCount)
+        {
+            var fst = GetFixedSizeTree(index);
+
+            totalCount = fst.NumberOfEntries;
+            if (afterValue == 0 || totalCount == 0)
+                return totalCount;
+
+            long count = 0;
+            using (var it = fst.Iterate())
```

Comment:
My problem is that you are trying to get accurate answer, and on large dbs, that can be problematic.
Consider the case of Stackoverflow db we are testing, 45GB in size, that will add a lot of overhead, and we can't afford it in stats endpoint that we'll query when the system is already under load.